syft/cataloging/relationships.go

```diff
@@ -0,0 +1,30 @@
+package cataloging
+
+type RelationshipsConfig struct {
+	FileOwnership                                 bool `yaml:"file-ownership" json:"file-ownership" mapstructure:"file-ownership"`
```

Comment:
`FileOwnership` and `FileOwnershipOverlap` don't seem to mean anything by themselves. What does it mean to say `file-ownership: true`? (I think it means, "Syft should report relationships of the type where a package owns a file" and `file-ownership-overlap: true` means, "Syft should report relationships where multiple packaages own a file?" but I'm not sure I'm right.)

Reply:
Ultimately this configuration holds which kinds of relationships should be captured or not. Happy for suggestions on better names!

> `file-ownership: true` ... means, "Syft should report relationships of the type where a package owns a file" 

correct 👍 

> `file-ownership-overlap: true` means, "Syft should report relationships where multiple packages own a file?"

correct 👍